web/src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+#prod db
+spring.datasource.url=jdbc:mysql://localhost:3306/tag_db
+spring.datasource.username=root
+spring.datasource.password=12345
+
+#dev db
+spring.second-datasource.url=jdbc:mysql://localhost:3306/test
+spring.second-datasource.username=root
+spring.second-datasource.password=12345
```

Comment:
It's forbidden to store any sensitive info (e.q. passwords, tokens) in public storage